configs/opa_bundles/default/authz.rego

```diff
@@ -6,29 +6,34 @@ import rego.v1
 # Helpers
 #
 
-is_active_org_member_of(org_id) if input.authn_user_orgs[org_id].status == "ACTIVE"
+has_active_role_in_org(user, org_id, "") if {
+	m := user.org_memberships[org_id]
+	m.status == "ACTIVE"
+}
 
-has_active_role_in_org(org_id, role) if {
-	org := input.authn_user_orgs[org_id]
-	org.status == "ACTIVE"
-	role in org.roles
+has_active_role_in_org(user, org_id, role) if {
+	m := user.org_memberships[org_id]
+	m.status == "ACTIVE"
+	role in m.roles
 }
 
-is_org_admin(org_id) if has_active_role_in_org(org_id, "admin")
+is_active_org_member_of(org_id) if has_active_role_in_org(input.authn_user, org_id, "")
+
+is_org_admin(org_id) if has_active_role_in_org(input.authn_user, org_id, "admin")
 
-is_active_member_of_resource_org := is_active_org_member_of(input.resource_org_id)
+is_active_member_of_resource_org := is_active_org_member_of(input.resource.org_id)
```

Comment:
"Resource" is too generic, it keeps confusing me when I try to read this file. Please rename to "subject" throughout this file.

Reply:
done